### PR TITLE
Remove circular deps

### PR DESCRIPTION
**Pull request by ineiti**
Mostly cleanup

### CODE REVIEW

File: external/js/cothority/src/network/connection.ts [hunk 1]
```diff
@@ -1,213 +1,10 @@
-import { Message, util } from "protobufjs/light";
```
[the remark this answers is not in the document]
Comment (ineiti) [reply external/js/cothority/src/network/connection.ts]: Cannot think of a better name. Either propose a better name, or we keep it like this.

[the remark this answers is not in the document]
Comment (tharvik) [reply external/js/cothority/src/network/connection.ts]: Rename it to `websocket-connections.ts`; or create a subdir "websocket" with all `websocket*.ts` moved there and basename stripped.

[the remark this answers is not in the document]
Comment (ineiti) [reply external/js/cothority/src/network/connection.ts]: moved `LeaderConnection` to `websocket.ts` and renamed to `rosterwsconnection.ts`.

**Comment by tharvik** [comment]
@ineiti travis fails in `test: js > cothority` with
```
spec/support/conondes.ts(7,36): error TS2307: Cannot find module '../../src/network/connection'.
```

**Pull request merged**